app/src/main/res/values/strings.xml

```diff
@@ -245,4 +245,18 @@
   <string name="switch_tabs">Switch tabs</string>
   <string name="smiling_face">:D</string>
   <string name="close_all_tabs">Close all tabs</string>
+  <string name="add_note">Add Note</string>
+  <string name="save">Save</string>
+  <string name="note">Note</string>
+  <string name="wiki_book_title">Wiki Book Title</string>
```

Comment:
ZIM File title?

Reply:
I was going for "Article Title" actually - Corrected in a new [commit](https://github.com/kiwix/kiwix-android/pull/1191/commits/54a94935c2834bb8679627925d70930c4e7c26eb)